hw/top_darjeeling/sw/dt/sim_dv.c

```diff
@@ -8,10 +8,8 @@
 
 static const uint32_t clock_freqs[kDtClockCount] = {
     [kDtClockMain] = 1 * 1000 * 1000 * 1000,
-    [kDtClockIo] = 1 * 1000 * 1000 * 1000,
+    [kDtClockIo] = 250 * 1000 * 1000,
```

Comment:
This change should be fixed up into the third commit?

Reply:
I squashed that to the previous commit that replaces the div_4 with the IO clock. The first one only removes the div_2 and uses div_4 instead.